src/pe_reports/report_generator.py

```diff
@@ -22,21 +44,73 @@ def load_template():
 
 def export_set(prs):
     """Export PowerPoint report set to output directory."""
-    pptx_out = "Customer_ID_Posture_Exposure.pptx"
-    prs.save(os.path.join(SAVE_PATH, pptx_out))
+    try:
+        pptx_out = "Customer_ID_Posture_Exposure.pptx"
+        prs.save(os.path.join("/output", pptx_out))
+    except FileNotFoundError as not_found:
+        logging.error(f"{not_found} : Missing input data. No report generated.")
```

Comment:
Mentioning the same thing in this file. Additionally for this line is the error message accurate?

```suggestion
        logging.error("%s : Missing input data. No report generated.", not_found)
```

Reply:
Resolved in 4d64c0f